tools/wptrunner/wptrunner/webdriver_server.py

```diff
@@ -84,6 +84,8 @@ def stop(self, force=False):
         self.logger.debug("Stopping WebDriver")
         if self.is_alive:
             return self._proc.kill()
+        if force:
```

Comment:
Will this not normally be unreached because of the previous branch, even when force is set?

Reply:
`.kill()` uses a SIGTERM signal. For cases where the subprocess is not responding this  could be still alive after kill() it.

Reply:
Right, but `self.is_alive` will be true so we'll return from the function at the previous line, no?

Reply:
oh. Shame on me. My eyed ignored the `return`. Of course I made a mistake writing the PR. I will rework it

Reply:
It's fine, it's easy to miss things. :)

Reply:
I just made the changes. Not sure if the temporary variable  matches with the code-style guides.

Side-note: this patch sends the `KILL` signal only if a `SIGTERM` has not success terminating the process.

Reply:
The unittest in the CI are failing but due to a build error in the `pip install` when it attempts to build `cffi` (https://paste.debian.net/1107425/) . Not sure @gsnedders not sure if I could force the  execution Taskcluster CI again. Forcing another push maybe?

Reply:
I think the easiest is to force a newer push. Seize that to edit the commit log and add the PR number (#19188) to the title.

Reply:
You can also close and reopen the PR. I'll do that now.

Reply:
CI looks green know. Thanks for the help.